### PR TITLE
Use uri for mongo sb517

### PR DESCRIPTION
**Pull request by JunAishima**
A couple of other changes
- remove creation of indexes from this code
- update versioneer

### CODE REVIEW

File: conftrak/test/conftest.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import ujson
+import jsonschema
+import uuid
+import sys
+import time as ttime
+import subprocess
+import contextlib
+from conftrak.client.commands import ConfigurationReference
+from conftrak.exceptions import ConfTrakNotFoundException
+from conftrak.ignition import Application
```
Comment (gwbischof) [inline conftrak/test/conftest.py:14]: Do we need all of these imports?

**Review by gwbischof — APPROVED** [review]

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -56,7 +53,7 @@ jobs:
         if: matrix.dependencies == 'conda'
         run: |
           set -vxeo pipefail
-          conda install -y -c conda-forge doct jsonschema mock pymongo pytest pyyaml requests tornado ujson
+          conda install -y -c conda-forge doct jsonschema mock mongomock pymongo pytest pytest-tornado==0.8.1 pyyaml requests tornado ujson
```
Comment (gwbischof) [inline .github/workflows/testing.yml:56]: I think we can remove pytest-tornado

File: .github/workflows/testing.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
```
Comment (gwbischof) [inline .github/workflows/testing.yml:15]: I think we can remove 3.6 and 3.7

File: conftrak/test/conftest.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        ps.terminate()
+
+
+@pytest.fixture(scope="session")
+def conftrak_server():
+    with conftrak_process() as conftrak_fixture:
+        yield
+
+
+@pytest.fixture
```
Comment (gwbischof) [inline conftrak/test/conftest.py:53]: I think this was needed by pytest-tornado, which you aren't using any more. So I think you can delete this.

File: .github/workflows/testing.yml (hunk 3 above)
Comment (JunAishima) [reply .github/workflows/testing.yml:15]: we are still using 3.6 for the LSDC GUI - one of the libraries requires that (albula for viewing diffraction images). fortunately, Shekar has found a way to extract that software into a separate process, so we should be able to release that requirement now.

File: .github/workflows/testing.yml (hunk 2 above)
Comment (JunAishima) [reply .github/workflows/testing.yml]: was able to get rid of this in https://github.com/NSLS-II/conftrak/pull/7/commits/5df5ec3d9129cc73ae3f81534ed9080e77034821

File: conftrak/test/conftest.py (hunk 1 above)
Comment (JunAishima) [reply conftrak/test/conftest.py]: no - removed in https://github.com/NSLS-II/conftrak/pull/7/commits/9d38ebc5017d928194c606b8ebafdac23580965d and https://github.com/NSLS-II/conftrak/pull/7/commits/0a330b914bb7b086592c1e3d8288ba1f010b5456

File: conftrak/test/conftest.py (hunk 4 above)
Comment (JunAishima) [reply conftrak/test/conftest.py]: removed in https://github.com/NSLS-II/conftrak/pull/7/commits/86ff07d1d874477f583fdd739b7e4f25af82f6a3

**Pull request merged**